api/v1/routes/user.py

```diff
@@ -98,6 +102,25 @@ async def change_password(
         message='Password changed successfully'
     )
 
+@user.get(path="/{user_id}", status_code=status.HTTP_200_OK)
+def get_user(user_id : str,
+             current_user : Annotated[User , Depends(user_service.get_current_user)],
+             db : Session = Depends(get_db)
+             ):
+    user = user_service.fetch(db=db , id=user_id)
+
+    return {
```

Comment:
return the right response. There is a success_response function. Please use that

Reply:
@joboy-dev  the success response doesnt follow the format of the issue it uses data and the issue that other teams are using is user